modules/integration/src/test/scala/scala/cli/integration/BspTestDefinitions.scala

```diff
@@ -149,6 +149,40 @@ abstract class BspTestDefinitions(val scalaVersionOpt: Option[String])
     }
   }
 
+  val importPprintOnlyProject = TestInputs(
+    Seq(
+      os.rel / "simple.sc" -> s"import $$ivy.`com.lihaoyi::pprint:${Constants.pprintVersion}`"
```

Comment:
No need to get this one via constants IMO… It's fine to hardcode dependency versions in the tests.

Reply:
I did to make it ease to track all the dependencies that are used during the build/testing. If we will use a tool to scan vulnerable dependencies in future, it won't find it if it;s hardcoded

Reply:
(but ofc I can revert it if you have a strong opinion)

Reply:
There's already many hardcoded dependencies in the tests… 

Let's either revert, or put test dependencies aside from the main ones, in the build.